gridcapa-cse-lib/data/src/main/java/com/farao_community/farao/cse/data/target_ch/TargetChDocument.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.farao_community.farao.cse.data.target_ch;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.transform.stream.StreamSource;
+import java.io.InputStream;
+import java.time.OffsetDateTime;
+import java.util.*;
+
+final class TargetChDocument {
+    private final TargetsDocument targetsDocument;
+
+    static TargetChDocument create(InputStream targetChInputStream) throws JAXBException {
+        TargetsDocument targetsDocument =  JAXBContext.newInstance(TargetsDocument.class)
```

Comment:
This block JAXBContext.newInstance(..).....getValue() is used in several classes (DailyNtcDocument class, YearlyNtcDocument class, TargetChDocument class) so it can be better if you extract this block in a generic method  to avoid code duplication.

Reply:
Done